segmentation_labeling_app/rois/rois.py

```diff
@@ -0,0 +1,240 @@
+
+sql_insert_roi = """ INSERT INTO rois_prelabeling (transform_hash,
+                     ophys_segmentation_commit_hash,
+                     creation_date, upload_date, manifest, experiment_id,
+                     roi_id) 
+                     VALUES (?, ?, ?, ?, ?, ?, ?) """
+
+
+class ROI:
+    """Class is used for manipulating ROI from LIMs for serving to labeling app
+
+    This class is used for loading the ROIs from LIMs DB tables and
+    contains pre processing methods for ROIs loaded from LIMs. These methods
+    are used to define drawing parameters as well as other ROI class
+    methods useful for post processing required to display to end user.
+    Attributes:
+        image_shape: the shape of the image the roi is contained within
+        experiment_id: the unique id for the segmentation run
+        roi_id: the unique id for the ROI in the segmentation run
+        _sparse_coo: the sparse matrix containing the probability mask
+        for the ROI
+    """
+
+    def __init__(self,
+                 coo_rows: Union[np.array, List[int]],
+                 coo_cols: Union[np.array, List[int]],
+                 coo_data: Union[np.array, List[float]],
+                 image_shape: Tuple[int, int],
+                 experiment_id: int,
+                 roi_id: int):
+        self.image_shape = image_shape
+        self.experiment_id = experiment_id
+        self.roi_id = roi_id
+        self._sparse_coo = coo_matrix((coo_data, (coo_rows, coo_cols)),
+                                      shape=image_shape)
+
+    @classmethod
+    def roi_from_query(cls, experiment_id: int,
+                       roi_id: int) -> "ROI":
+        """
+        Queries and builds ROI object by querying LIMS table for
+        produced labeling ROIs.
+        Args:
+            experiment_id: Id of the segmentation run
+            roi_id: Unique Id of the ROI to be loaded
+
+        Returns: ROI object for the given segmentation_id and roi_id
+        """
+        label_vars = query_utils.get_labeling_env_vars()
+
+        shape = query_utils.query(
+            f"SELECT * FROM public.segmentation_runs WHERE id={experiment_id}",
+            user=label_vars.user,
+            host=label_vars.host,
+            database=label_vars.database,
+            port=label_vars.port,
+            password=label_vars.password)[0]['video_shape']
+
+        roi = query_utils.query(
+            f"SELECT * FROM public.rois WHERE "
+            f"segmentation_run_id={experiment_id} AND id={roi_id}",
+            user=label_vars.user,
+            host=label_vars.host,
+            database=label_vars.database,
+            port=label_vars.port,
+            password=label_vars.password)
+        return ROI(coo_rows=roi[0]['coo_row'],
+                   coo_cols=roi[0]['coo_col'],
+                   coo_data=roi[0]['coo_data'],
+                   image_shape=shape,
+                   experiment_id=experiment_id,
```

Comment:
I think we're going to want to propogate the `ophys_experiment_id` through the pipeline. Here, you're propogating the `segmentation_run_id` which are different things. Have a look at the table designs again:
https://github.com/AllenInstitute/informatics_design_docs/tree/master/ophys/segmentation#table-designs